packages/widgets/src/components/PieChart/PieChartLegend.tsx

```diff
@@ -28,23 +28,34 @@ export const PieChartLegend = ({
                 as={'span'}
                 css={markerStyles}
               />
-              <Typography variant="h6">
-                {typeof renderLabel === 'function' ? renderLabel(item) : label}
-              </Typography>
+              {item.customLegendItem ? (
+                (item.customLegendItem as JSX.Element)
+              ) : (
+                <Typography variant="h6">
+                  {typeof renderLabel === 'function'
+                    ? renderLabel(item)
+                    : label}
+                </Typography>
+              )}
             </li>
           );
         })}
       </PieChartLegendList>
       <PieChartLegendList css={valueListStyles}>
-        {data.map((item) => (
-          <li key={`subtitle-${item.id}`}>
-            <Typography variant="subtitle" color={theme.colors.greyDarker60}>
-              {typeof renderValue === 'function'
-                ? renderValue(item)
-                : item.value}
-            </Typography>
-          </li>
-        ))}
+        {data.map(
```

Comment:
As we discussed, we don't need additional ul element (PieChartLegendList). Please add additional property to the component (ex.: "variant") and use its value for the output

Reply:
done